.idea/.gitignore

```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```

Comment:
We don't want this file committed. We should git ignore all of the .idea directory.

Reply:
yeah, missed that...fixed.

Reply:
It is still in this commit though.

Reply:
Ok, folder removed from the repo.